res/css/structures/_ToastContainer.pcss

```diff
@@ -141,7 +141,7 @@ Please see LICENSE files in the repository root for full details.
         }
 
         .mx_Toast_description {
-            max-width: 272px;
+            max-width: 366px;
```

Comment:
This toast wants to be wider, so we need a bigger toaster.

Reply:
@element-hq/design